From 54435cb671f26fb17a35b795c38262ba58166bdc Mon Sep 17 00:00:00 2001
From: Donald McIntyre <29529586+TokenHash@users.noreply.github.com>
Date: Fri, 6 Mar 2020 13:25:13 -0700
Subject: [PATCH 1/3] Fixed url capitalization - added language to include
 controversies
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

• ECIP -> ecip

• Added language "including, but not limited to" in the phrase that clarifies that devs/editors can change ECIPs to previous statuses in the case in the examples but also if other major events occur, such as the recent ProgPow controversy after EIP-1057 was moved to accepted. The text modified was:

"However, core developers and editors can decide to move ECIPs back to previous statuses after the fact, or to `Rejected` or `Withdrawn` status, at their discretion provided there is reasonable cause, ***including but not limited to,*** a major, but correctable, flaw was found in the ECIP, a major but not correctable flaw was found in the ECIP."
---
 _specs/{ECIP-0001.md => ecip-0001.md} | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
 rename _specs/{ECIP-0001.md => ecip-0001.md} (99%)

diff --git a/_specs/ECIP-0001.md b/_specs/ecip-0001.md
similarity index 99%
rename from _specs/ECIP-0001.md
rename to _specs/ecip-0001.md
index 476e7602..e4667345 100644
--- a/_specs/ECIP-0001.md
+++ b/_specs/ecip-0001.md
@@ -203,7 +203,7 @@ For Meta or Informational ECIPs:
 WIP -> DRAFT -> LAST CALL -> ACTIVE
 ````
 
-However, core developers and editors can decide to move ECIPs back to previous statuses after the fact, or to `Rejected` or `Withdrawn` status at their discretion provided there is reasonable cause, e.g. a major, but correctable, flaw was found in the ECIP, or a major but not correctable flaw was found in the ECIP.
+However, core developers and editors can decide to move ECIPs back to previous statuses after the fact, or to `Rejected` or `Withdrawn` status, at their discretion provided there is reasonable cause, including but not limited to, a major, but correctable, flaw was found in the ECIP, a major but not correctable flaw was found in the ECIP.
 
 Champions of an ECIP may decide on their own to change the status between `Draft`, `Deferred`, or `Withdrawn`.
 

From 00fd8bd64cdb601f9260fbbda9a2179c30d30a9d Mon Sep 17 00:00:00 2001
From: Donald McIntyre <29529586+TokenHash@users.noreply.github.com>
Date: Sat, 7 Mar 2020 10:30:02 -0700
Subject: [PATCH 2/3] Update ecip-0001.md

---
 _specs/ecip-0001.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/_specs/ecip-0001.md b/_specs/ecip-0001.md
index e4667345..8f1c3bca 100644
--- a/_specs/ecip-0001.md
+++ b/_specs/ecip-0001.md
@@ -203,7 +203,7 @@ For Meta or Informational ECIPs:
 WIP -> DRAFT -> LAST CALL -> ACTIVE
 ````
 
-However, core developers and editors can decide to move ECIPs back to previous statuses after the fact, or to `Rejected` or `Withdrawn` status, at their discretion provided there is reasonable cause, including but not limited to, a major, but correctable, flaw was found in the ECIP, a major but not correctable flaw was found in the ECIP.
+However, core developers and editors can decide to move ECIPs back to previous statuses after the fact, or to `Rejected` or `Withdrawn` status, at their discretion provided there is reasonable cause, including but not limited to, a major, but correctable, flaw was found in the ECIP, or a major but not correctable flaw was found in the ECIP.
 
 Champions of an ECIP may decide on their own to change the status between `Draft`, `Deferred`, or `Withdrawn`.
 

From b97d5045fa06e694d053060eec4cd310c9f32049 Mon Sep 17 00:00:00 2001
From: Donald McIntyre <29529586+TokenHash@users.noreply.github.com>
Date: Fri, 13 Mar 2020 10:42:07 -0600
Subject: [PATCH 3/3] Adding editor

As there is no rough consensus in the ecosystem debates of removing him, I added Wei Tang (@Sorpaas) to the editor list.
---
 _specs/ecip-0001.md | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/_specs/ecip-0001.md b/_specs/ecip-0001.md
index 8f1c3bca..64dbf491 100644
--- a/_specs/ecip-0001.md
+++ b/_specs/ecip-0001.md
@@ -79,11 +79,12 @@ A good reason to transfer ownership is because the original author no longer has
 
 The current ECIP editors are:
 
+* Zachary Belford (@BelfordZ)
 * Mr. Meows D. Bits (@meowsbits)
 * Cody Burns (@realcodywburns)
 * Talha Cross (@soc1c)
 * Yaz Khoury (@YazzyYaz)
-* Zachary Belford (@BelfordZ)
+* Wei Tang (@sorpaas)
 
 ## ECIP Editor Responsibilities & Workflow
 
